product_docs/docs/pgd/5/raft/03_migrating_to_raft_subgroups.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Migrating to Raft subgroups
+---
+
+This section assumes that you have an already running cluster and wish to introduce Raft subgroups to your cluster.
+
+
+## Migrating to Raft subgroups (without TPA)
+
+A number of configuration steps need to be taken to enable Raft subgroups to an existing cluster.
+
+* A “top-level” group needs to be defined for all nodes in the PGD cluster. Use bdr.create_node_group
+* A subgroup needs to be created for each location. Use bdr_create_node_group
+* Each location’s subgroup should be added to the “top-level” group. Use bdr.join_node_group(), bdr.switch_node_group() or bdr.alter_node_group().
```

Comment:
I don't think you can change the parent of a group. Instead, it needs to be specified to `bdr.create_node_group` (note the dot vs underscore) as argument `parent_group_name` in the step above.

Also note that there is no `alter_node_group`, but two variants: `alter_node_group_option` and `alter_node_group_config`. None of the two allow you to change the parent, IIRC.

`bdr.join_node_group` joins a node to a group, whereas `bdr.switch_node_group` allows you to move a node from one group to another. None of these two change the group hierarchy.